topi/recipe/conv/test_conv_int8_intel.py

```diff
@@ -112,8 +112,8 @@ def run_inference(data_dtype, kernel_dtype, out_dtype, im_height, im_width, in_f
 
     with tvm.target.create(TARGET_NAME):
         conv = topi.nn.conv2d_NCHWc(data, kernel, stride=hstride,
-                                    padding=hpad, layout='NCHWc',
-                                    out_layout='NCHWc', out_dtype=out_dtype)
+                                    padding=hpad, dilation=(1, 1),
```

Comment:
is dilation=(1,1) not already a default value in the `topi.nn.conv2d_NCHWc` API? if not should it be?

Reply:
This is a low-level topi operator and not the Relay operator. Relay operator has dilation default arguments which will be passed on to TOPI, if dilation was not provided in the framework graph. So, I think not having a default here is fine.